helicopyter.py

```diff
@@ -75,13 +75,16 @@ def push(
 
 # ruff: noqa: T201
 def multisynth(
-    all_or_conas_or_paths: Iterable[str], *, hashicorp_configuration_language: bool = False
+    all_or_conas_or_paths: Iterable[str],
+    *,
+    change_directory: Path | None = None,
+    hashicorp_configuration_language: bool = False,
 ) -> None:
     if not all_or_conas_or_paths:
         print('No codenames specified. Doing nothing.')
         return
 
-    top_directory = Path(__file__).parent
+    top_directory = change_directory if change_directory else Path.cwd()
```

Comment:
If change_directory is empty string that's identical to cwd here, right?

Reply:
Yes, passing an empty string will trigger the current working directory default, which I think is reasonable.

```
$ python -m helicopyter -C '' foundation
Generating deploys/foundation/terraform/main.tf.json
Pushing github_repository.airdjang
Pushing github_repository.allowedflare
Pushing github_repository.helicopyter
```